lib/src/facts/resolver.cc

```diff
@@ -24,6 +26,23 @@ namespace facter { namespace facts {
                 throw invalid_name_pattern_exception(ex.what());
             }
         }
+
+#ifdef LEATHERMAN_USE_LOCALES
+        // build Accept-Language list for HTTP-based resolvers
```

Comment:
Do we really need to do this for all resolvers? I'd probably make this a construct-on-demand field (do the work in http_langs if _http_langs is empty).